### PR TITLE
Add AWS cost estimates to Single Cluster Reference Architecture

### PR DESCRIPTION
**Pull request by lucasvaltl**
## Description
Adds AWS cost estimates to the Single cluster reference architecture - GCP estimates are still in progress. 

Also added an estimate as to how many workspaces this may allow you to run, based on [this comment](https://gitpod.slack.com/archives/C01KLC56NP7/p1660154736599959?thread_ts=1660151211.174839&cid=C01KLC56NP7) (this ignores throughput and IOPS)

## Related Issue(s)

Fixes #

## How to Review
Please review the linked [cost estimates](https://calculator.aws/#/estimate?id=dba87a5a0ccb84ae1445eb38c77c170ddc7a819a) (linked from netlify link ---> docs>self-hosted>installation guides > single cluster ref arch) and ensure that:
- These estimates cover everything that costs $$$ in AWS that gets created when this reference architecture is created
- The usage values here make sense:
  - LB traffic: 0.1 GB per hour
  - S3 usage for images & workspace backups: 500mb per image or backup, 200 images and 800 backups per month - 5TB of S3 usage

## Release Notes

```release-note
NONE
```

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2559"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2559"></a>

### CODE REVIEW

**Comment by lucasvaltl** [comment]
[Internal discussion](https://gitpod.slack.com/archives/C01KLC56NP7/p1660154736599959?thread_ts=1660151211.174839&cid=C01KLC56NP7) about what information we can give that helps answer the question of "what scale do I get for this price"

**Review by adrienthebo — APPROVED** [review]
One small comment - the RDS line item is set to consume 20GB/month and I can see database consumption growing larger than that. Bumping this to 200GB/month is only $20 more. I defer to you if this warrants adjustment.

Otherwise, LGTM!

File: gitpod/docs/self-hosted/latest/reference-architecture/single-cluster-ref-arch.md [hunk 1]
```diff
@@ -22,10 +22,11 @@ title: "Single-Cluster Reference Architecture"
...
+| Intended for:   | Continous usage of Gitpod at a company-wide scale in a reliable way by leveraging popular cloud provider services such as S3 and RDS.                                                                                                                                                                                              |
+| Limitations:    | - Is bound to a single cluster. Deploying in several regions requires setting up several Gitpod installations <br /> - Requires external dependencies (Object storage, registry, database) to be set up                                                                                                                            |
+| Cost Estimates: | - [AWS\*](https://calculator.aws/#/estimate?id=dba87a5a0ccb84ae1445eb38c77c170ddc7a819a) <br /> \* Cost estimates do not include egress - however, we typically see egress cost to be an additional 15% on top of the estimates here. These estimates are just examples - exact cost will depend on your set up and usage profile. |
```
Comment (lucasvaltl) [inline gitpod/docs/self-hosted/latest/reference-architecture/single-cluster-ref-arch.md:29]:
```suggestion
| Cost Estimates: | - [AWS\*](https://calculator.aws/#/estimate?id=258a343acea8193acd54ffabf1a61e203389439a) <br /> \* Cost estimates do not include egress - however, we typically see egress cost to be an additional 15% on top of the estimates here. These estimates are just examples - exact cost will depend on your set up and usage profile. |
```

**Comment by lucasvaltl** [comment]
Approval from @mrzarquon [here](https://gitpod.slack.com/archives/C01KLC56NP7/p1660220214760439?thread_ts=1660151211.174839&cid=C01KLC56NP7) (internal)

**Comment by lucasvaltl** [comment]
Added an internal [document](https://www.notion.so/gitpod/Reference-Architecture-Cost-Estimates-b34589d1df2349a6b6a7fe50a06e333f#9c7c3d7233b4429eb4c933722a3728b6) to notion that details the assumptions made for this cost estimate.

**Pull request merged**